doc/api/crypto.md

```diff
@@ -2949,6 +2949,171 @@ Does not perform any other validation checks on the certificate.
 
 ## `node:crypto` module methods and properties
 
+### `crypto.argon2(algorithm, parameters, callback)`
+
+<!-- YAML
+added: REPLACEME
```

Comment:
Lets not forget to get this modified with this PR number

Reply:
added: is replaced with the Node.js version, not a PR link

Reply:
ah thats right